src/Compilers/CSharp/Portable/CommandLine/CSharpCommandLineParser.cs

```diff
@@ -1489,6 +1501,7 @@ public new CSharpCommandLineArguments Parse(IEnumerable<string> args, string? ba
                 RuleSetPath = ruleSetPath,
                 OutputDirectory = outputDirectory!, // error produced when null
                 DocumentationPath = documentationPath,
+                GeneratedFilesDirectory = generatedFilesDirectory,
```

Comment:
```suggestion
                GeneratedFilesOutputDirectory = generatedFilesDirectory,
```

Just to match the other "output directories"?

Reply:
(also better matches the MSBuild target, not that there's a strict requirement to do so)